docs/script/wildfly-cli/embed-server/configure-managed-executor.adoc

```diff
@@ -1 +1,2 @@
-|MANAGED_EXECUTOR_SERVICE_CORE_THREADS |managed-executor-service.core-thread |16 | Maximum number of Wildfly managed executors. (If it was not configured, t would be dynamic - [number of cpu cores] x 2)   
+|MANAGED_EXECUTOR_SERVICE_CORE_THREADS |managed-executor-service.core-thread |16 | Minimum number of Wildfly managed executors. (If it was not configured, t would be dynamic - [number of cpu cores] x 2)
+|MANAGED_EXECUTOR_SERVICE_GRPC_CORE_THREADS |managed-executor-service.grpc.core-thread |16 | Minimum number of Wildfly managed executors for gRPC service calls. (If it was not configured, t would be dynamic - [number of cpu cores] x 2)
```

Comment:
van benne egy typo, bar az eredetileg masolt sorban is volt :)

Reply:
Javítottam az eredetiben is :)